shuffle/cli/src/transactions.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use crate::shared::{get_home_path, Home};
+use anyhow::{anyhow, Result};
+use diem_types::account_address::AccountAddress;
+use reqwest::{Client, Response};
+use serde_json::Value;
+use std::{cmp::max, fs, io, io::Write, str::FromStr, thread, time};
+use url::Url;
+
+// Will list the last 10 transactions and has the ability to block and stream future transactions.
+pub async fn handle(network: Url, tailing: bool) -> Result<()> {
+    let home = Home::new(get_home_path().as_path())?;
+    let address_str = fs::read_to_string(home.get_latest_address_path())?;
+    let address = AccountAddress::from_str(address_str.as_str())?;
+    let client = reqwest::Client::new();
+
+    let account_seq_num = get_account_sequence_number(&client, &network, address).await?;
+    let mut old_seq_num = max(account_seq_num as i64 - 10, 0);
```

Comment:
Could you remove L38 and the else block by setting the 2nd parameter to -1?

```rust
let mut old_seq_num = max(account_seq_num as i64 - 10, -1);
```

Reply:
nit: let's call it `prev_seq_num`

Reply:
Earlier, I tried setting the second param to -1 which would pass in -1 as the start parameter of GET account_transactions, but this results in an error.